### ISSUE TITLE
[A11y] Desktop/Mobile Work Show Page-CPD Child: Downloads > Metadata Only CSV

### ISSUE THREAD

**Issue by azle** [OregonDigital/OD2#1954]
### Operating System

MacOS

### Browser

Chrome

### Affected page

Work Show

### Admin user?

No

### Issue Description

Work Show Page-CPD Child: https://staging.oregondigital.org/concern/documents/df727057n?locale=en

As a non logged in user and click on Download and go to MetaData Only (CSV). I get the "ACK" msg. However, I tried it again and I cannot repeat the error until I refresh the page.

### Can anybody QA or does this need an expert?

Anybody

**Comment by CGillen** [comment]
Fixed somewhere else?

**Comment by azle** [comment]
@CGillen I revalidated and found 1 error see screen snap:

<img width="824" alt="Screen Shot 2022-02-02 at 8 57 43 AM" src="https://user-images.githubusercontent.com/8353860/152200556-6a9f0db7-a41d-40bc-a26e-47c4ca4fac3c.png">

**Comment by CGillen** [comment]
Above issue is related to an UV issue we currently can not fix. Please disregard until we can edit the UV iframe (if we decide to)
